guides/doc-Administering_Red_Hat_Satellite/topics/proc_integrating-satellite-with-red-hat-single-sign-on-for-external-authentication.adoc

```diff
@@ -218,13 +218,23 @@ For more information about group mappers, see https://access.redhat.com/document
 +
 . Open the `_{Keycloak-short}.example.com_/auth/realms/_{Keycloak-short}_REALM_/.well-known/openid-configuration` URL and note the values to populate the options in the following steps.
 +
-. Set the value for the Open IDC audience:
+. Add the value for the Hammer client in the Open IDC audience. 
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# hammer settings set --name oidc_audience \
+--value "['_{foreman-example-com}_-hammer-openidc']"
+----
 +
+[NOTE]
+====
+If you have already registered another client with keycloak, make sure also to append that audience in the array. For example:
```

Comment:
```suggestion
If you register several {Keycloak} clients to {Project}, ensure to append all audiences in the array. For example:
```
1. Use the attribute {Keycloak}.
2. Use one-word `ensure` instead of the phrasal verb `make sure`.
3. Remove also.
4. Use a present tense.

Reply:
"make sure to" is a set phrase. "ensure to" is incorrect. I've reworded slightly and applied the change.